README.md

```diff
@@ -1660,12 +1660,22 @@ A rule that assembles data into a tarball which can be use as in `layers` attr i
       <td><code>empty_dirs</code></td>
       <td>
         <code>List of directories, optional</code>
-        <p>Directory to add to the layer.</p>
+        <p>Directory to remove in the layer.</p>
```

Comment:
Can you please revert this change? I believe `empty_dirs` doesn't remove anything in the layer, but rather creates empty dirs as stated in the description.

Reply:
I don't know how I messed this diff up so badly. Fixing it now.